pandas/tests/indexing/test_multiindex.py

```diff
@@ -1,19 +1,47 @@
 from datetime import datetime
-from warnings import catch_warnings
+from warnings import catch_warnings, simplefilter
 
 import numpy as np
+from numpy.random import randn
 import pytest
 
+from pandas.compat import (
+    StringIO, lrange, lzip, product as cart_product, range, zip)
 from pandas.errors import PerformanceWarning, UnsortedIndexError
 
 import pandas as pd
 from pandas import (
-    DataFrame, Index, MultiIndex, Panel, Period, Series, Timestamp, date_range,
-    period_range)
+    DataFrame, Index, MultiIndex, Panel, Period, Series, Timestamp, concat,
+    date_range, period_range, read_csv)
+import pandas.core.common as com
 from pandas.tests.indexing.common import _mklbl
 from pandas.util import testing as tm
 
 
+@pytest.fixture
+def frame():
```

Comment:
i had left some comments on the fixture names

Reply:
sorry @jreback , either i'm not looking at the right comment or i've misunderstood something. can you clarify?

Reply:
yes, can you give these meaningful names